ark/utils/deepcell_service_utils.py

```diff
@@ -37,25 +39,39 @@ def create_deepcell_output(deepcell_input_dir, deepcell_output_dir, fovs=None,
             ValueError:
                 Raised if there is some fov X (from fovs list) s.t.
                 the file <deepcell_input_dir>/fovX.tif does not exist
-        """
+    """
+
+    # extract all the files from deepcell_input_dir
+    input_files = io_utils.list_files(deepcell_input_dir, substrs='.tif')
+
+    # set fovs equal to input_files it not already set
     if fovs is None:
-        tifs = io_utils.list_files(deepcell_input_dir, substrs='.tif')
-        fovs = io_utils.extract_delimited_names(tifs, delimiter='.')
+        fovs = input_files
+
+    # now extract only the names of the fovs without the file extension
+    fovs = io_utils.extract_delimited_names(fovs, delimiter='.')
 
+    # make sure that all fovs actually exist in the list of input_files
+    misc_utils.verify_in_list(
+        fovs=fovs,
+        deepcell_input_files=[in_file.replace('.tif', '') for in_file in input_files])
+
```

Comment:
I think you can just do:
```
in_fovs = io_utils.extract_delimited_names(tifs, delimiter='.')
...
misc_utils.verify_in_list(
    fovs=fovs,
    deepcell_input_files=in_fovs
)
```
Regardless, currently, if someone has `fov1.tiff`, the `replace` line will turn that into `fov1f` not `fov1`.